bin/node/src/chain_spec.rs

```diff
@@ -11,6 +13,13 @@ use sp_application_crypto::key_types;
 use sp_consensus_aura::sr25519::AuthorityId as AuraId;
 use sp_core::{ed25519, sr25519, Pair, Public};
 use sp_runtime::traits::{IdentifyAccount, Verify};
+use std::{env::VarError, fmt::Display, str::FromStr};
+
+const SESSION_PERIOD_ENV_VAR: &str = "SESSION_PERIOD";
+const MILLISECS_PER_BLOCK_ENV_VAR: &str = "MILLISECS_PER_BLOCK";
+
+const FAUCET_HASH: [u8; 32] =
+    hex!("eaefd9d9b42915bda608154f17bb03e407cbf244318a0499912c2fb1cd879b74");
```

Comment:
is this always hardcoded or could be a configurable variable in itself?

Reply:
I'll leave it hardcoded for now. There is no benefit in having it configurable for now, and making it specifiable when starting nodes is nontrivial: the hex macro which we use right now works only with string literals, not any values.